dts/bindings/sensor/invensense,mpu6050.yaml

```diff
@@ -16,3 +16,36 @@ properties:
       The INT signal default configuration is active-high.  The
       property value should ensure the flags properly describe the
       signal that is presented to the driver.
+
+  accel-fs:
+    type: int
+    default: 2
+    description: |
+      Accelerometer full-scale range.
+      An X value represents a range of +/- X g.
+    enum:
+      - 2
+      - 4
+      - 8
+      - 16
+
+  gyro-fs:
+    type: int
+    default: 250
+    description: |
+      Gyroscope full-scale range.
+      An X value represents a range of +/- X deg/s.
+    enum:
+      - 250
+      - 500
+      - 1000
+      - 2000
+
+  smplrt-div:
+    type: int
+    default: 0
```

Comment:
Could you add in description if the default is the power-on reset value?

Reply:
done